connector.rb

```diff
@@ -529,17 +535,17 @@
           nil,
           input['query']
         )
-        documents = parsed_query&.[](:documents)
-        if documents.present?
-          operation = documents.find { |doc| doc[:operation_name] == input['operation_name'] }
-        end
+        docs = parsed_query&.[](:documents)
+        operation = docs.find { |doc| doc[:operation_name] == input['operation_name'] } || docs.first if docs.present?
         if operation.present?
-          call(
+          output_fiels = call(
```

Comment:
@klaasvandeweerdt #nit but can we fix the typo here?

Reply:
@bennettgo, I updated it to `output_fields`. I also fixed another small label issue. The field `Rate limit` of the object `Rate limit` is now called `Limit` instead of `Rate limit` as the object itself is already called `Rate limit`.